metadata/reportssql/drug_functions.sql

```diff
@@ -335,3 +335,67 @@ BEGIN
     RETURN result;
 END$$
 DELIMITER ;
+
+-- drugOrderIsARefill
+
+DROP FUNCTION IF EXISTS drugOrderIsARefill;
```

Comment:
@tmvumbi2 I don't see this function being used anywhere?

Reply:
@lauravignoli the requirement changed leading to this function not to be used; it was left in the code just in case we need it in the future